extensions/wikia/AdEngine/js/slot/scrollHandler.js

```diff
@@ -39,9 +45,10 @@ define('ext.wikia.adEngine.slot.scrollHandler', [
 			log('Scroll event listener has been added', 'debug', logGroup);
 			for (var slotName in config) {
 				if (config.hasOwnProperty(slotName)) {
-					if (config[slotName].hasOwnProperty('reloadedViewMax') &&
+					if (config[slotName].trigger.match(/^scroll\.(top|bottom)$/) === null || (
```

Comment:
I would rather move `config[slotName].trigger.match(/^scroll\.(top|bottom)$/)` to separate `if` for better readability.